barretenberg/cpp/src/barretenberg/honk/proof_system/protogalaxy_prover.hpp

```diff
@@ -264,7 +264,7 @@ template <class ProverInstances_> class ProtoGalaxyProver_ {
                 // Accumulate the i-th row's univariate contribution
                 accumulate_relation_univariates(thread_univariate_accumulators[thread_idx],
                                                 extended_univariates[thread_idx],
-                                                relation_parameters,
+                                                instances.relation_parameters,
```

Comment:
Hm, I don't really see why this is confusing. I'm open to suggestions though.

Reply:
as I mentioned, my mental model was that relation parameters will be folded in protogalaxy prover. I don't feel super strongly about this though so if you think this approach is better because you found some problems/ugliness with having it at prover level, i'm okay with it

Reply:
I think what it's missing is a comment to make it clear relation parameters have been already folded (because they have to be before this stage) so this is not interpreted as if the relation parameters we should use are accumulator's ones :-?